website/docs/docs/core/connect-data-platform/profiles.yml.md

```diff
@@ -54,6 +63,37 @@ This section identifies the parts of your `profiles.yml` that aren't specific to
 
 </File>
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.8">
+
+<File name='profiles.yml'>
+
+```yml
+
+<profile-name>:
+  target: <target-name> # this is the default target
+  outputs:
+    <target-name>:
+      type: <bigquery | postgres | redshift | snowflake | other>
+      schema: <schema_identifier>
+      threads: <natural_number>
+
+      ### database-specific connection details
+      ...
+
+    <target-name>: # additional targets
+      ...
+
+<profile-name>: # additional profiles
+  ...
+
+```
+
+</File>
+
+</VersionBlock>
+
 ## User config
 
-You can set default values of global configs for all projects that you run using your local machine. See the docs on [global configs](/reference/global-configs/about-global-configs) for details.
+You can set default values of global configs for all projects that you run using your local machine. Refer to [About global configs](/reference/global-configs/about-global-configs) for details.
```

Comment:
Fast-follow: we should consider adding versioning block(s) here since there will no longer way to set configs/flags across all projects after this is deprecated.

Reply:
Within that fast-follow (or separately), we should consider separating the code example for `config:` within `profiles.yml` entirely. That might make the versioning blocks cleaner and easier to read and maintain.

Reply:
Raised in issue https://github.com/dbt-labs/docs.getdbt.com/issues/4996